problems/numberOfDigits.js

```diff
@@ -7,8 +7,13 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    str = n.toString()
+    let count = 0
+    for(let i = 0; i < str.length; i++){
+        count++
```

Comment:
Which string property will also allow you to get the number of characters in the string? How can you solve this problem without a for loop?